### PR TITLE
Presentation: Nested content value label full-stack tests

### PR DESCRIPTION
**Pull request by Yato333**
Part of https://github.com/iTwin/presentation/issues/654
imodel-native: https://github.com/iTwin/imodel-native/pull/811

### CODE REVIEW

File: full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+              ],
+            },
+          ],
+        },
+        ...additionalRules,
+      ],
+    };
+  }
+
+  it("each related content item has display label", async () => {
```
Comment (grigasp) [inline full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts:104]:
```suggestion
  it("returns related content items with display label", async () => {
```

File: full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+      keys: new KeySet(),
+    }))!;
+
+    for await (const item of items) {
+      for (const value of Object.values(item.values)) {
+        if (!Value.isNestedContent(value)) {
+          continue;
+        }
+        expect(value.length).to.eq(2);
+        expect(value[0].labelDefinition).to.deep.eq(EMPTY_LABEL);
```
Comment (grigasp) [inline full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts:119]: why not check labels of both related content values?

File: full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+      }
+    }
+  });
+
+  it("applies correctly class name label override", async () => {
```
Comment (grigasp) [inline full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts:124]:
```suggestion
  it("applies class name label override on nested content values", async () => {
```

File: full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+          continue;
+        }
+        expect(value.length).to.eq(2);
+        expect(value[0].labelDefinition?.displayValue).to.eq("Color");
+        expect(value[1].labelDefinition?.displayValue).to.eq("Color");
+      }
+    }
+  });
+
+  it("applies correctly composite label override", async () => {
```
Comment (grigasp) [inline full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts:150]:
```suggestion
  it("applies composite label override on nested content values", async () => {
```

File: full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts (hunk 2 above)
Comment (Yato333) [reply full-stack-tests/presentation/src/frontend/content/ContentDisplayLabels.test.ts:119]: Fixed

**Review by grigasp — APPROVED** [review]

**Pull request merged**